.github/workflows/ci.yml

```
@@ -54,7 +54,7 @@ jobs:
         env:
           JULIA_DISTRIBUTED_TESTING_STANDALONE: 1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v5
         with:
           file: lcov.info
```

This was renamed to `files` in v5:
```suggestion
          files: lcov.info
```